src/prefect/cli/orion.py

```diff
@@ -124,6 +125,8 @@ async def start(
                 run_process,
                 command=[
                     "uvicorn",
+                    "--app-dir",
+                    str(Path(prefect.__file__).parent.parent),
```

Comment:
We have some constants in `__init__.py` to make this clearer
```

# The absolute path to this module
__module_path__ = pathlib.Path(__file__).parent
# The absolute path to the root of the repository
__root_path__ = __module_path__.parents[1]
# The absolute path to the built UI within the Python module
__ui_static_path__ = __module_path__ / "orion" / "ui"
```

This path is just going to be `site-packages` when a non-editable install is used. Is this the intention?

Reply:
Yep, it just needs to point at site-packages, uvicorn overrides the python path to `.` so we kind of just need to point it back in the right direction.

Reply:
I updated it to use `__root_path__`.

Reply:
Sorry I've mislead you here. `__root_path__` is only valid during development usage. We should clarify that in a command. You want `__module_path__.parent` but root path is another level up because we need to step out of `src` when the repository is being used.

Reply:
Ha, no worries. Updated again and added a comment to the `__root_path__`.